docs/docs/configuration/pages.md

```diff
@@ -95,7 +95,17 @@ export default function SignIn({ providers }) {
 }
 
 export async function getServerSideProps(context) {
-  const providers = await getProviders()
+  const { req } = context;
+  const session = await getSession({ req });
```

Comment:
Let's use `getServerSession`. Will also need the import as it seems to be missing: 
https://github.com/ChinonsoIg/next-auth/blob/0cef6f0eac11259eb8946c8f40ff68ea61262729/docs/docs/configuration/pages.md?plain=1#L81

https://next-auth.js.org/configuration/nextjs#getserversession

Reply:
Is `getServerSession` stable enough.
I tried it on my local environment and got this error message, "Module not found: Can't resolve 'v8'".
I tried resolving it, but no headway yet.

Reply:
Yes, it has been for months, we recently removed the `unstable_` flag. The way it worked was always stable, I just wasn't sure about the API, so we kept the rights to change it. But since it has worked well in production for a long time, we could just remove the prefix.

You got that `v8` error because you were using it in the wrong context. `getServerSession` - as the name suggests - is meant for the server, and you were trying to use it somewhere in the browser. You could open an issue for that, but still, in this PR, we should use `getServerSession`.

Reply:
Ooh, I get it now.